cms/static/sass/views/_bulkupdate.scss

```diff
@@ -0,0 +1,89 @@
+// Studio - Course Utilities - BulkUpdate
+// ====================
+
+.view-bulkupdate {
+
+  .content-primary, .content-supplementary {
```

Comment:
Do you know if there are any common/shared CSS files we could include that already provide this basic styling?